### PR TITLE
chore/learner record auth

### PR DESCRIPTION
**Pull request by superafroman**

### CODE REVIEW

File: src/lib/xapi.ts [hunk 1]
```diff
@@ -140,7 +141,7 @@ export async function record(
 			type = Type.Event
```
Comment (tav) [inline src/lib/xapi.ts]: Should this be `Type.Elearning` now?

File: src/ui/controllers/xapi.ts [hunk 2]
```diff
@@ -12,7 +13,7 @@ export async function proxy(ireq: express.Request, res: express.Response) {
 
 	const agent = {
 		account: {
-			homePage: 'https://cslearning.gov.uk',
+			homePage: xapi.HomePage,
```
Comment (tav) [inline src/ui/controllers/xapi.ts]: Nice catch!

File: src/lib/xapi.ts (hunk 1 above)
Comment (superafroman) [reply src/lib/xapi.ts]: Good spotting!

File: src/ui/controllers/xapi.ts (hunk 2 above)
Comment (superafroman) [reply src/ui/controllers/xapi.ts]: Wouldn't have noticed but the missing trailing slash meant xapi didn't return results matching the user for these statements 🙄

**Pull request merged**